src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -1407,26 +1407,25 @@ public boolean handleTokenExpiration(HttpServletRequest httpRequest, HttpServlet
         }
 
         User user = User.get2(authentication);
+        if (user == null) {
```

Comment:
given the check on the `authentication` above this would IIUC indicate a bug in Jenknis, so throwing the exception would I think be preferable (to identify "something really really bad has happened).

If we are not using `authentication` in this method apart from in the above call, then we can remote the null/anonymous check for it in the 3 lines above and then this null check of `User` would be correct.

Reply:
Since `authentication` is not used, I removed the null/anonymous check 3 lines above